modules/machine-config-node-disruption.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machine-config-node-disruption_{context}"]
+= Understanding node restart behaviors upon machine config changes
+
+By default, when you make certain changes to the fields in a `MachineConfig` object, the Machine Config Operator (MCO) drains and reboots the nodes associated with that machine config. However, you can create a _node disruption policy_ that defines a set of changes to some Ignition config objects that would require little or no disruption to your workloads.
+
+A node disruption policy allows you to define the configuration changes that cause a disruption to your cluster, and which changes do not. This allows you to reduce node downtime when making small machine configuration changes in your cluster. To configure the policy, you modify the `MachineConfiguration` object, which is in the `openshift-machine-config-operator` namespace. See some example node disruption policies in the example `MachineConfiguration` objects that follow.
+
+[NOTE]
+====
+There are machine configuration changes that always require a reboot, regardless of any node disruption policies. For more information, see _About the Machine Config Operator_.
+====
+
+After you create the node disruption policy, the MCO validates the policy to search for potential issues in the file, such as problems with formatting. The MCO then merges the policy with the cluster defaults and populates the `status.nodeDisruptionPolicyStatus` fields in the machine config with the actions to be performed upon future changes to the machine config. The configurations in your policy always overwrite the cluster defaults.
+
+[IMPORTANT]
+====
+The MCO does not validate whether a change can be successfully applied by your node disruption policy. As such, you are responsible to ensure the accuracy of your node disruption policies.
+====
+
+For example, you could configure a node disruption policy so that sudo configurations do not require a node drain and reboot. Or, you could configure your cluster so that updates to `sshd` are applied with only a reload of that one service.
```

Comment:
```suggestion
For example, you can configure a node disruption policy so that sudo configurations do not require a node drain and reboot. Or, you can configure your cluster so that updates to `sshd` are applied with only a reload of that one service.
```